libs/core/src/lib/verification/Verification.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useRef } from 'react';
+import { View, TextInput, StyleSheet } from 'react-native';
+
+import { Palette } from '@bmg-react-native-nx/constants';
+import { Typography } from '@bmg-react-native-nx/constants';
```

Comment:
ერთ import-ში გავაერთიანოთ
```suggestion
import { Palette, Typography  } from '@bmg-react-native-nx/constants';
```